docs/source/channel_update_tutorial.rst

```diff
@@ -333,7 +303,7 @@ protobufs. This command will output a new protobuf binary named ``org3_update.pb
 
 .. code:: bash
 
-  configtxlator compute_update --channel_id $CHANNEL_NAME --original config.pb --updated modified_config.pb --output org3_update.pb
+  configtxlator compute_update --channel_id channel1 --original config.pb --updated modified_config.pb --output org3_update.pb
```

Comment:
It may be worth doing this inside a `config` folder given how many artifacts that it leaves around...

Reply:
Agreed, but it would also be worth doing for the create channel tutorial as well. I can create a separate JIRA for that.

Reply:
Ah, nevermind, the create channel tutorial uses a `channel-artifacts` folder. I'll update this to follow that pattern.

Reply:
Done